src/components/tag/Tags.tsx

```diff
@@ -63,7 +63,7 @@ const TagsPage = (props: InjectedProps) => {
     <>
       <div className="container w-100">
         <FileUploader
-          fileSizeLimit={60000}
+          csvKeyCountLimit={100}
```

Comment:
That variable must not be hard-coded. It is easy to miss in a React tree structure and it is part of the core business logic. So a config constant or an environment variable would be better options.

Reply:
Corrected it @olivierobert. I normally create a constants for such magic numbers and strings. Seems I missed this one. 😅